tests/html/css/selectize.css

```diff
@@ -0,0 +1,323 @@
+/**
```

Comment:
I thought about using a cdn or external link so that these files didn't have to go in the repo. I'm not sure if that would mess with test reliability, however, so I have included them in the tests/html directory. Any other suggestions?